client/src/pages/database/index.tsx

```diff
@@ -152,6 +176,9 @@ export const Database = () => {
                   placeholder={selectedApp}
                   isSearchable={false}
                   aria-labelledby="app-select-dropdown"
+                  noOptionsMessage={() =>
```

Comment:
should we just remove the select in such case? Not sure which one is the best

Reply:
I will try both approaches and see which looks better. I guess it depends on how it "feels" when you unlink db and see the select appear again. If that looks ok, then removing select and showing some text in case of no items makes sense.